test/tests/preconditioners/fdp/tests

```diff
@@ -6,4 +6,11 @@
     max_parallel = 1
     deleted = '#5153'
   [../]
+  [./test_standard]
```

Comment:
The other test is deleted with a ticket number. Did you happen to look at that? Just curious.